scripts/install-dev.sh

```diff
@@ -146,13 +146,23 @@ show_important_note() {
 
 has_python() {
   "$1" -c '' >/dev/null 2>&1
-  if [ "$?" -eq 127 ]; then
+  if [ "$?" -ne 0 ]; then
     echo 0
   else
     echo 1
   fi
 }
 
+install_indygreg_python() {
+  url="https://github.com/indygreg/python-build-standalone/releases/download/20230116/cpython-3.11.1+20230116-$INDYGREG_ARCH-$INDYGREG_PLATFORM-install_only.tar.gz"
+  cwd=$(pwd)
+  mkdir -p python-runtime && cd python-runtime
+  show_info "Downloading and installing standalone Python 3.11.1..."
```

Comment:
Is there a reason you specified the python version directly without using a variable?

Reply:
I think it's just the latest version available at the time of writing. The version requirement is from what we do in `install-dev.sh`, and this is separate from what Backend.AI itself needs.

Reply:
You've got it right. Additionally most of the jobs `$bpython` is to execute standalone python scripts (which does not require state-of-the-art python versions) and thus I think it is okay to just leave the version hard-coded rather than being customizable (either by user or maintainer).